docs/modules/deployment/pages/core/system-requirements.adoc

```diff
@@ -31,16 +31,11 @@ OpenNMS {page-component-title} requires one of the following operating systems:
 
 [options="autowidth"]
 |===
-| Operating System  | Compatible Versions (64-bit)
+| Operating System  | Compatible Versions (Intel x86 64 bit)
 
-| RHEL
-| {compatible-rhel7} +
-{compatible-rhel8} +
-{compatible-rhel9}
-
-| CentOS
-| {compatible-centos7} +
-{compatible-centos-stream}
+| RHEL / Rocky Linux / Alma Linux
```

Comment:
+ CentOS Stream

Reply:
The reason I've picked those was, I got this from people familiar from support.
What do you think about CentOS is more like a rolling release and probably harder test if the setup procedure works and will probably break more often?

I personally don't have a strong opinion here.

Reply:
Stream is the only thing I run in my lab environment, so it gets as much internal testing as any other distro, I imagine :)